python/lsst/ip/diffim/utils.py

```diff
@@ -711,3 +732,354 @@ def plotWhisker(results, newWcs):
+                     extent=extent)
+    plt.title(title)
+    if showBars:
+        plt.colorbar(fig, cmap='gray')
+    return fig
+
+
+def displayImage(image, showBars=True, width=8, height=2.5):
+    """Use matplotlib.pyplot.imshow() to display an afw.image.Image within
+    its bounding box.
+
+    @see displayImages()
+    """
+    return displayImages((image), showBars=True, width=8, height=2.5)
+
+
+def displayImages(images, showBars=True, width=8, height=2.5):
+    """Use matplotlib.pyplot.imshow() to display up to three
+    afw.image.Images alongside each other, each within its
+    bounding box.
+
+    @see display2dArray() for params not listed here.
+    @param images tuple of up to three images to display
+    @param image The image to display
+    @param width The width of the display (inches)
+    @param height The height of the display (inches)
+
+    @return matplotlib.pyplot.figure dispaying the image
+
+    """
+    plt = importMatplotlib()
+    if not plt:
+        return
+
+    fig = plt.figure(figsize=(width, height))
+    for i, image in enumerate(images):
+        bbox = image.getBBox()
+        extent = (bbox.getBeginX(), bbox.getEndX(), bbox.getBeginY(), bbox.getEndY())
+        plt.subplot(1, len(images), i+1)
+        ma = image.getArray()
+        display2dArray(ma, title='Data', showBars=showBars, extent=extent)
+    return fig
+
+
+def displayMaskedImage(maskedImage, showMasks=True, showVariance=False, showBars=True, width=8,
+                       height=2.5):
+    """Use matplotlib.pyplot.imshow() to display a afwImage.MaskedImageF,
+    alongside its masks and variance plane
+
+    @see displayImage() for params not listed here
+
+    @param maskedImage MaskedImage to display
+    @param showMasks Display the MaskedImage's masks
+    @param showVariance Display the MaskedImage's variance plane
+
+    @return matplotlib.pyplot.figure dispaying the image
+
+    """
+    plt = importMatplotlib()
+    if not plt:
+        return
+
+    fig = plt.figure(figsize=(width, height))
+    bbox = maskedImage.getBBox()
+    extent = (bbox.getBeginX(), bbox.getEndX(), bbox.getBeginY(), bbox.getEndY())
+    plt.subplot(1, 3, 1)
+    ma = maskedImage.getArrays()
+    display2dArray(ma[0], title='Data', showBars=showBars, extent=extent)
+    if showMasks:
+        plt.subplot(1, 3, 2)
+        display2dArray(ma[1], title='Masks', showBars=showBars, extent=extent)
+    if showVariance:
+        plt.subplot(1, 3, 3)
+        display2dArray(ma[2], title='Variance', showBars=showBars, extent=extent)
+    return fig
+
+
+def displayExposure(exposure, showMasks=True, showVariance=False, showPsf=False, showBars=True,
+                    width=8, height=2.5):
+    """Use matplotlib.pyplot.imshow() to display a afw.image.Exposure,
+    including its masks and variance plane and optionally its Psf.
+
+    @see displayMaskedImage() for params not listed here
+
+    @param exposure Exposure to display
```

Comment:
I suggest you list the types here, not in the brief description. There's more room in the @params